asciidoc/courses/app-python/course.adoc

```diff
@@ -1,9 +1,10 @@
-= Building Neo4j Applications with Python
-:categories: developer:2, python, software-development:15, intermediate:15,  development:5
+= Building a real-world Neo4j Application with Python
```

Comment:
Does this title have enough differentiation from  Building Neo4j Applications with Python ?